eng/pipelines/configuration/TSAConfig.gdntsa

```diff
@@ -11,9 +11,5 @@
     "projectName": "DevDiv",
     "areaPath": "DevDiv\\NET Tools\\Project",
     "iterationPath": "DevDiv",
-    "tools": [
-        "CredScan",
-        "PoliCheck",
-        "APIScan"
-    ]
+    "allTools": true
```

Comment:
This simply grabs whatever Guardian tool results are available instead of having to specify each tool individually.